spec/views/catalog/_exhibits_document_header_default.html.erb_spec.rb

```diff
@@ -17,7 +17,10 @@
 
   describe 'IIIF Drag-n-Drop Icon' do
     context 'when the document has a manifest' do
-      let(:document) { SolrDocument.new(iiif_manifest_url_ssi: 'https://purl.stanford.edu/bc853rd3116/iiif/manifest') }
+      let(:document) do
+        SolrDocument.new(iiif_manifest_url_ssi: 'https://purl.stanford.edu/bc853rd3116/iiif/manifest',
+                         content_metadata_type_ssm: %w(image))
+      end
```

Comment:
@drh-stanford could you add a case for when the document has a manifest but does not have a valid `content_metadata_type_ssm`?

Reply:
Ok, I added that test.